lib/modules/manager/nuget/util.spec.ts

```diff
@@ -59,22 +57,20 @@ describe('modules/manager/nuget/util', () => {
 
       const registries = await getConfiguredRegistries('NuGet.config');
       expect(registries?.length).toBe(2);
-      expect(registries![0].name).toBe('nuget.org');
-      expect(registries![0].url).toBe('https://api.nuget.org/v3/index.json');
-      expect(registries![0].sourceMappedPackagePatterns).toEqual(['*']);
-
-      expect(registries![1].name).toBe('contoso.com');
-      expect(registries![1].url).toBe('https://contoso.com/packages/');
-      expect(registries![1].sourceMappedPackagePatterns).toEqual([
-        'Contoso.*',
-        'NuGet.Common',
-      ]);
+      expect(registries![0]).toEqual({
+        name: 'nuget.org',
+        url: 'https://api.nuget.org/v3/index.json',
+        sourceMappedPackagePatterns: ['*'],
+      });
+      expect(registries![1]).toEqual({
+        name: 'contoso.com',
+        url: 'https://contoso.com/packages/',
+        sourceMappedPackagePatterns: ['Contoso.*', 'NuGet.Common'],
+      });
```

Comment:
don't change existing tests if not needed, otherwise combine this to a single `toEqual` or `toMatchObject`

Reply:
I didn't plan to change existing tests first, but [this job](https://github.com/renovatebot/renovate/actions/runs/11907985528/job/33182852346) forced me to do so. I don't know why it didn't report an error on the previous commits. 🤷